connectors/src/lib/dpdf2text.ts

```diff
@@ -0,0 +1,33 @@
+import { spawn } from "child_process";
+
+export async function dpdf2text(pdfPath: string): Promise<string> {
+  const args: string[] = ["-layout", "-enc", "UTF-8", pdfPath, "-"];
```

Comment:
It's a common way of telling the process to output to stdout instead of a file, when the argument is a file path.
In this case, it's implemented by the binary `pdftotext`.

Reply:
Got it, thanks 🙏